src/uucore/src/lib/parser/parse_time.rs

```diff
@@ -49,16 +51,21 @@ pub fn from_str(string: &str) -> Result<Duration, String> {
     if len == 0 {
         return Err("empty string".to_owned());
```

Comment:
I think we should return `invalid time interval {}` here too, for consistency with coreutils.